### ISSUE TITLE
Number entry fields unreadable on default theme

### ISSUE THREAD

**Issue by dreua** [gitahead/gitahead#270]
The contents of the number entry fields are not visible when using the default theme. In the dark theme all is fine. Is this a problem specific to my environment / distribution?
The "Fetch every x minutes" setting (Tab: General) is also affected.

Just compiled from source on Fedora 30:
GitAhead v2.5.10 - Montag, 16. September 2019 - 82c6c97

![default theme screenshot](https://user-images.githubusercontent.com/15114715/64990591-60c79300-d8d0-11e9-8f39-33d6db8ee9dd.png)

**Comment by hackhaslam** [comment]
What window manager? Do you have a non-default scale factor? Does the scaling look broken elsewhere?

**Comment by dreua** [comment]
Oh sorry: gnome 3.32
Scaling is default (100%) and no broken scaling elsewhere that I am aware of.

**Comment by hackhaslam** [comment]
Hmm... okay. I haven't seen anything like that, but I can try to reproduce.

**Comment by dreua** [comment]
I ran some more tests on a VM running Fedora 32 / rawhide (i.e. cutting edge / unstable) with the same results. The v2.5.10 tag didn't build but it's the same with the [next commit fixing build errors](https://github.com/gitahead/gitahead/commit/2c8ccb5890b1059bba6147d06c2a28faa0a7d75e).

When I tried the binary distribution, surprisingly the problem did not occur. Must be a difference in the build environment then...
